javascript/javascript1/week4/task_1.js

```diff
@@ -0,0 +1,86 @@
+let myName = "";
+let todo = [];
+
+function getReply(command) {
+  const array = command.split(" ");
+  let response="";
+  if (command.includes("hello my name is")) {
+    myName = array[4];
+    response = `nice to see you ${myName}`;
+  } else if (command.includes("what is my name")) {
+    response = `my name is ${myName}`;
+  } else if (command.includes("add fishing")) {
+    todo.push("fishing");
+    response = "added fishing to todo";
+  } else if (command.includes("add singing")) {
+    todo.push("singing");
+    response = "added singing to todo";
+  } else if (command.includes("remove fishing")) {
+    todo.shift();
+    response = "removed fishing from todo";
+  } else if (command.includes("what is on ")) {
+    response = todo;
+  } else if (command.includes("what day ")) {
+    const today = new Date();
+    const weekday = [
+      "Sunday",
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday"
+    ];
+    const dayOfWeek = weekday[today.getDay()];
+    const months = [
+      "January",
+      "February",
+      "March",
+      "April",
+      "May",
+      "June",
+      "July",
+      "August",
+      "September",
+      "October",
+      "November",
+      "December"
+    ];
+    const curMonth = months[today.getMonth()];
+    const curYear = today.getFullYear();
+    const dateOfMonth = today.getDate();
+    response = `today is ${dayOfWeek} , ${dateOfMonth} of ${curMonth} , ${curYear}`;
+  } else if (command.includes("What is ")) {
```

Comment:
Good job with the conditional! You can also add an "else" at the end to catch any other possible outcomes as errors or wrong user inputs. 🤩